controllers/apply/awards-for-all/fields.js

```diff
@@ -1221,15 +1258,33 @@ module.exports = function fieldsFor({ locale, data = {}, flags = {} }) {
         organisationTradingName: {
```

Comment:
Like above worth converting this to use a field type if we're updating this field.